src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1731,29 +1731,29 @@
   <data name="MakeMethodDeclaredOnImplementationTypeStaticMayProduceInvalidCode" xml:space="preserve">
     <value>Converting an instance method to a static method may produce invalid code</value>
   </data>
-  <data name="DerivesFromPreviewClassMessage" xml:space="preserve">
-    <value>'{0}' derives from preview class '{1}' and therefore needs to opt into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
+  <data name="DerivesFromPreviewClassMessageWithCustomMessagePlaceholder" xml:space="preserve">
+    <value>'{3}' '{0}' derives from preview class '{1}' and therefore needs to opt into preview features. See '{2}' for more information.</value>
```

Comment:
Just making sure, do you want the message added within a quote?

Not sure if there is a way to inform the localization team that the placeholder '{3}' is not included in the sentence and should not be moved elsewhere in the sentence. Not sure the `locked` parameter could work for this `<comment>{Locked="{3}"}</comment>`, there might be another parameter exist for this case